### PR TITLE
Remove custom-submodule from vc_util

### PR DESCRIPTION
**Pull request by przydatek**
Remove `vc_util::custom::*`, as the corresponding functionality is now provided by a generic validation function `validate_ii_presentation_and_claims`.  Update `vc-util-js` accordingly.

### CODE REVIEW

File: src/vc_util_js/src/lib.rs [hunk 1]
```diff
@@ -20,10 +22,19 @@ pub fn validate_verified_adult_presentation(
         issuer_canister_id: Principal::from_slice(issuer_canister_id),
         issuer_origin: String::from_utf8(issuer_origin.to_vec()).expect("wrong issuer origin"),
     };
-    vc_util::custom::validate_verified_adult_presentation(
+    let credential_spec = {
```
Comment (frederikrothenberger) [inline src/vc_util_js/src/lib.rs:25]: So this function will be removed as well at some point? And allow to pass through the credential spec from JS? Or is `vc_util_js` never intended to become the JS equivalent of `vc_util`?

**Review by frederikrothenberger — APPROVED** [review]
LGTM, thanks!

File: src/vc_util_js/src/lib.rs (hunk 1 above)
Comment (przydatek) [reply src/vc_util_js/src/lib.rs:25]: Yes, the function will be replaced with the generic validator that takes a spec.  But the js module is not indented to be fully equivalent to `vc_util`, at least for now.

**Comment by przydatek** [comment]
Thanks!

**Pull request merged**